corehq/form_processor/backends/sql/dbaccessors.py

```diff
@@ -470,7 +470,7 @@ def get_attachment_content(form_id, attachment_name, stream=False):
 
     @staticmethod
     def get_form_operations(form_id):
-        return list(XFormOperationSQL.objects.plproxy_raw('SELECT * from get_form_operations(%s)', [form_id]))
```

Comment:
Would it make sense to remove the SQL function definitions too? In this case that would be
- [corehq/sql_accessors/sql_templates/get_form_operations.sql](https://github.com/dimagi/commcare-hq/blob/975fcca431cc4cc5ba1f2be7f9aeef393c9fa6d2/corehq/sql_accessors/sql_templates/get_form_operations.sql)
- [corehq/sql_proxy_accessors/sql_templates/get_form_operations.sql](https://github.com/dimagi/commcare-hq/blob/975fcca431cc4cc5ba1f2be7f9aeef393c9fa6d2/corehq/sql_proxy_accessors/sql_templates/get_form_operations.sql)

Reply:
Nevermind. I see you did that in https://github.com/dimagi/commcare-hq/pull/26465